.gemini.yml

```diff
@@ -0,0 +1,25 @@
+rootUrl: http://127.0.0.1:3000/test/visual
+gridUrl: http://127.0.0.1:4444/wd/hub
+screenshotsDir: ./test/visual/screens
+
+system:
+  plugins:
+    express:
+      port: 3000
+    sauce: true
+
+browsers:
```

Comment:
Please add the Safari block to the config and comment it. Even better if you add some explanation comment beside so in future we will know what was the issue and it will be easier to revisit.